openedx/core/djangoapps/credentials/tests/test_signals.py

```diff
@@ -119,9 +119,8 @@ def test_send_grade_records_enabled(self, _mock_is_course_run_in_a_program, mock
         mock_send_grade_to_credentials.delay.reset_mock()
 
         # Correctly not sent
-        site_config.site_values['ENABLE_LEARNER_RECORDS'] = False
```

Comment:
This is new functionality, so I think the lines that were deleted are still a valid test situation, and we just want to add an additional test.  Could be a separate unit test, or, maybe the deleted lines could be added at the end and re-verified?

Note: Programs has a similar test and I imagine we'd want a parallel change? See https://github.com/edx/edx-platform/blob/151bd136667d27bf85256e382ed5b5ea0144e00f/openedx/core/djangoapps/programs/tests/test_signals.py#L167

Reply:
Right, I'll revert the modified test and add a new one below this one. However I'm not keen on adding tests elsewhere, as I'd rather avoid adding redundant unit tests.